pkg/e2e/operators/operators.go

```diff
@@ -391,7 +391,7 @@ Loop:
 			return err
 		default:
 			if elapsed < timeoutDuration {
-				log.Printf("Waiting %v for %s clusterRoleBinding to exist", (timeoutDuration - elapsed), clusterRoleBindingName)
+				log.Printf("Waiting %v for %s clusterRoleBinding to exist", timeoutDuration-elapsed, clusterRoleBindingName)
```

Comment:
Could we change this back to having the space? 
Removing the space between variables makes it hard to read, I thought it was one word.

Reply:
My IDE's auto reformat must've done it - will do